### PR TITLE
Add tooltips to display name and email

### PR DESCRIPTION
**Pull request by onehappycat**
#18158 Add tooltips to ~~username~~ 'display name' and 'email' fields in user management.

### CODE REVIEW

**Comment by kesselb** [comment]
Thanks :+1: 

Would you mind to share a screenshot? ;)

**Comment by onehappycat** [comment]
> Thanks 👍
> 
> Would you mind to share a screenshot? ;)

Screenshot 2019-12-02 at 19 45 08

**Review by skjnldsv — CHANGES_REQUESTED** [review]
Hum, this will conflict with @GretaD's work on the new layout #17239 
I would rather have this wait until the other to ease the potential rebase :)

**Comment by GretaD** [comment]
yes please lets wait, not that much left to do on my ticket. I'll ping you again again.

**Comment by onehappycat** [comment]
Sounds good. The new layout looks amazing. 👍

**Comment by blizzz** [comment]
@GretaD's PR was merged. What to do with this one?

**Comment by skjnldsv** [comment]
rework it to add the tooltips at the correct place :)

**Comment by onehappycat** [comment]
Rebased to current master, added the tooltips to correct places at the new layout.

Before:
Screenshot 2019-12-14 at 14 31 28

After:
Screenshot 2019-12-14 at 14 31 36

**Review by skjnldsv — APPROVED** [review]

File: apps/settings/src/components/UserList/UserRowSimple.vue [hunk 1]
```diff
@@ -15,12 +15,18 @@
 		<div class="name">
 			{{ user.id }}
 			<div class="displayName subtitle">
-				{{ user.displayname }}
+				<div v-if="user.displayname.length>20" v-tooltip.auto="user.displayname">
```
Comment (skjnldsv) [inline apps/settings/src/components/UserList/UserRowSimple.vue]: Hum, you could also just ellipsis with css and enable the tooltip only when the length is bigger than '20 char' ?

Comment (skjnldsv) [reply apps/settings/src/components/UserList/UserRowSimple.vue]: Also, we're using the three dot ellipsis character, not three real dots: `…`
And with a space before :)

**Review by skjnldsv — CHANGES_REQUESTED** [review]
See comments ;)

**Comment by onehappycat** [comment]
Rebased and added the full output of `make build-js-production`, but webpack is still complaining.

**Comment by skjnldsv** [comment]
@onehappycat something else was merged, please rebase again :p
After rebase you can use the `/compile amend /` command to trigger a compile of any js file by our bot ;)

**Review by skjnldsv — APPROVED** [review]

File: apps/settings/src/components/UserList/UserRowSimple.vue [hunk 2]
```diff
@@ -15,12 +15,16 @@
 		<div class="name">
 			{{ user.id }}
 			<div class="displayName subtitle">
-				{{ user.displayname }}
+				<div v-tooltip="user.displayname.length > 20 ? user.displayname : ''" class="cellText">
```
Comment (ChristophWurst) [inline apps/settings/src/components/UserList/UserRowSimple.vue]: is there any other way to detect the truncated text? the magic number 20 doesn't care about different fonts, letter width and dynamic size of the conainer.

Comment (skjnldsv) [reply apps/settings/src/components/UserList/UserRowSimple.vue]: Yes, use a vue ellipsis component/directive :)
In the meantime, I'd say let get this in, worst thing that could happen is a tooltip :see_no_evil:

**Review by ChristophWurst — APPROVED** [review]
I like happy cats

**Pull request merged**